core/actions/table.ts

```diff
@@ -114,6 +119,7 @@ const IBigQueryOptionsProperties = () =>
     "labels",
     "partitionExpirationDays",
     "requirePartitionFilter",
+    "withConnection",
```

Comment:
Hi @Ekrekr !
I am trying to add an option to the BigQuery config.
I added a new value to the config, but I am encountering a compilation error.
I am not sure what is causing it. Could you help me figure it out?

For reference, the following commands were successful:

```
bazel build cli  
bazel test //core/...  
```

Reply:
It's not your fault - we have a couple issues with our CI/CD which we're resolving currently.

I'll get this PR in once we've managed to fix them!

Reply:
Thank you for the clarification! I understand the issue is on the CI/CD side.

Please let me know if there's anything I can do to help in the meantime. I’ll wait for the fix and look forward to the PR being merged.

Thanks again for your support!